lib/helper/ArticleData.dart

```diff
@@ -22,127 +21,109 @@ class _ArticleDataState extends State<ArticleData> {
-                            crossAxisAlignment: CrossAxisAlignment.start,
-                            children: <Widget>[
-                              (widget.data[index]['urlToImage'] == null)
-                                  ? ClipRRect(
-                                      borderRadius: BorderRadius.circular(10.0),
-                                      child: Image(
-                                          image: NetworkImage(
-                                              'https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcRpRknF9IAeIQ2QCUaQ9JdQWoJUVUMVhXetvA&usqp=CAU')),
-                                    )
-                                  : ClipRRect(
-                                      borderRadius: BorderRadius.circular(10.0),
-                                      child: Image(
-                                        image: NetworkImage(
-                                            widget.data[index]['urlToImage']),
-                                      ),
-                                    ),
-                              Expanded(
-                                child: Container(
-                                  height: responsiveTitleHeight(mediaQuery),
-                                  child: Text(
-                                    widget.data[index]['title'] == null
-                                        ? 'Breaking News'
-                                        : widget.data[index]['title'],
-                                    style: GoogleFonts.merriweather(
-                                      textStyle: TextStyle(
-                                          letterSpacing: .4,
-                                          fontSize: 18.0,
-                                          fontWeight: FontWeight.w600),
-                                    ),
-                                  ),
-                                ),
-                              ),
-                              Expanded(
-                                child: Container(
-                                  child: Text(
-                                    widget.data[index]['description'] == null
-                                        ? ''
-                                        : widget.data[index]['description'],
-                                    style: GoogleFonts.merriweather(
-                                      textStyle: TextStyle(
-                                          letterSpacing: .2,
-                                          fontSize: 14.0,
-                                          fontWeight: FontWeight.w400),
-                                    ),
+      child: ListView.builder(
+        physics: const AlwaysScrollableScrollPhysics(),
+        itemCount: widget.data.length,
+        itemBuilder: (context, index) {
+          return GestureDetector(
+            onTap: () {
+              mediaQuery.size.width > 700
+                  ? _launchURL(widget.data[index]['url'])
```

Comment:
You can remove this ternary operator . This was supposed to be used with the web version .